components/select/select-top-control.component.ts

```diff
@@ -72,7 +72,7 @@ import { NzSelectItemInterface, NzSelectModeType, NzSelectTopControlItemType } f
           [disabled]="disabled"
           [value]="inputValue!"
           [autofocus]="autofocus"
-          [showInput]="true"
+          [showInput]="open && showSearch"
```

Comment:
is any reason to add `open` needed here?

Reply:
Prevent typing if autofocus is true

Reply:
this would disable search in all multiple select
here is the demo https://preview-5871-ng-zorro-antd.surge.sh/components/select/en#components-select-demo-size